src/scss/base/_typography.scss

```diff
@@ -60,13 +72,18 @@ h6,
 .epsilon {
     color: $color-headings;
     margin: 0;
-    font-family: $font-family-headings;
-    font-weight: $font-weight-headings;
     margin-bottom: 0;
+    font-family: $font-family-headings;
+    font-weight: $font-weight-base;
 
     small {
         font-weight: normal;
     }
+
+    .is-fontsLoading--heading & {
+        font-family: $font-family-preload !important;
+        font-weight: $font-weight-bold;
```

Comment:
Don't think this one is needed? Should pick up the weight from the styles above?